settings.gradle.kts

```diff
@@ -28,13 +29,13 @@ dependencyResolutionManagement {
 val isCI = System.getenv("CI") != null
 val skipBuildscan = System.getenv("SKIP_BUILDSCAN").toBoolean()
 gradleEnterprise {
+  server = "https://ge.opentelemetry.io"
```

Comment:
Previously scan was published only when building from ci but this check is now removed. Does this mean that scan is uploaded even for local builds?

Reply:
That is Gradle's suggestion, yes. The goal is to better understand builds (especially build failures) on local machines. If we are concerned about privacy we may change that to publish scans only on failures.

Reply:
I think it's too surprising if OSS builds phone home without an opt-in. Eventually someone will notice and complain hard :P

Reply:
Ok, will change to publish only on CI or if opted in